docs/howto/add-taxon-restrictions.md

```diff
@@ -1,6 +1,6 @@
 # Adding taxon restrictions
 
-Before adding taxon restrictions, please review the [types of taxon restrictions documentation](../explanation/taxon-constraints-explainer/#Types-of-Taxon-Restrictions).
+Before adding taxon restrictions, please review the [types of taxon restrictions documentation](https://oboacademy.github.io/obook/explanation/taxon-constraints-explainer/?h=taxon+restric#types-of-taxon-restrictions).
```

Comment:
```suggestion
Before adding taxon restrictions, please review the [types of taxon restrictions documentation](../explanation/taxon-constraints-explainer/#types-of-taxon-restrictions).
```
```suggestion
Before adding taxon restrictions, please review the [types of taxon restrictions documentation](https://oboacademy.github.io/obook/explanation/taxon-constraints-explainer/?h=taxon+restric#types-of-taxon-restrictions).
```

We should never use absolute linked to OBOOK resources; I changed it to a relative link, so lets see if it works.